sty/presentation/presentation.dtx

```diff
@@ -625,6 +626,7 @@
 % greater than the current precedence specified by |\pres@current@precedence|
 %    \begin{macrocode}
 \def\PrecWrite#1{\ifnum\pres@p>\pres@current@precedence\else{#1}\fi}
+\def\PrepostPrecWrite#1{\ifnum\pres@p@key>\pres@infty@minusone\else{#1}\fi}

```

Comment:
First, I can't just copy ``\PrecWrite`` into ``\prefix`` macro, because ``\pres@p`` is defined inside ``\prep@keys@mi``, which is a initialization macro only for ``mixfix``, not for ``prefix`` nor ``postfix``. So ``\pres@p`` is actually undefined when I only define a prefix operation.

Second, ``\PrepostPrecWrite`` only deals with ``nobrackets`` option. It won't change the precedence logic we had before. As we discussed on Monday, We still prefer the brackets around a prefix operator, if precedence level is not ``infty``, i.e, ``nobrackets``. 

Some sample outputs:
``\nminval{\set{a,b,c}}`` will generate ``min{a, b, c}``. (nobrackets option)
``\minval{a, b, c}`` will generate ``min(a, b, c)``. (p = 900)

Is that clear?